security/tpm/devdoc/tpm.md

```diff
@@ -0,0 +1,60 @@
+# azure-iot-security-tpm.TpmSecurityClient Requirements
+
+## Overview
+`TpmSecurityClient` provides query and key operations for the device registration client.
+
+## Example usage
+TBD
+
+### TpmSecurityClient(registrationId?: string, customTpm?: any) [constructor]
+
+The `TpmSecurityClient` constructor initializes a new instance of a `TpmSecurityClient` object that is used to query TPM entities and perform key manipulation and setup.
+
+**SRS_NODE_TPM_SECURITY_CLIENT_06_001: [** The `registrationId` argument if present will be returned as the `registrationId` for subsequent calls to `getRegistrationId`. **]**
+
+**SRS_NODE_TPM_SECURITY_CLIENT_06_002: [** The `customTpm` argument, if present` will be used at the underlying TPM provider.  Otherwise the TPM provider will the tss TPM client with a parameter of `false` for simulator use. **]**
+
+### getEndorsementKey(callback: (err: Error, endorsementKey: string) => void): void
+
+Queries and returns the public portion of the endorsement key in the TPM hardware.
+
+**SRS_NODE_TPM_SECURITY_CLIENT_06_006: [** The `getEndorsmentKey` function shall query the TPM hardware and return the `endorsementKey` in the callback. **]**
+
+**SRS_NODE_TPM_SECURITY_CLIENT_06_007: [** Any errors from interacting with the TPM hardware will cause in SecurityDeviceError to be returned in the err parameter of the callback. **]**
```

Comment:
>SecurityDeviceError [](start = 107, length = 19)

nit: backquotes around class names and argument names

Reply:
Done

---
In reply to: [160280767](https://github.com/Azure/azure-iot-sdk-node/pull/185#discussion_r160280767) [](ancestors = 160280767)